modules/rostests/kmtests/kmtest_drv/kmtest_fsminifilter.c

```diff
@@ -337,7 +339,7 @@ FilterInstanceSetup(
     UNICODE_STRING VolumeName;
     ULONG ReportedSectorSize = 0;
     ULONG SectorSize = 0;
-    NTSTATUS Status;
+    NTSTATUS Status = STATUS_FLT_DO_NOT_ATTACH;
```

Comment:
{{
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(347,8) :  warning: variable 'Status' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
    if (!(Flags & TESTENTRY_NO_INSTANCE_SETUP))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(398,11) :  note: uninitialized use occurs here
    return Status;
           ^~~~~~
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(347,4) :  note: remove the 'if' if its condition is always true
    if (!(Flags & TESTENTRY_NO_INSTANCE_SETUP))
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(340,19) :  note: initialize the variable 'Status' to silence this warning
    NTSTATUS Status;
                   ^
                    = 0
}}

Reply:
Not sure what the intent is with TESTENTRY_NO_INSTANCE_SETUP, since we have no tests using it yet. Would need Ged's input on whether Don't Attach or Success is better.

Reply:
Success would be better here. 
The test framework will always register for volume connections, but the caller has the choice to ignore being notified with the TESTENTRY_NO_INSTANCE_SETUP. With this change, if the caller chooses to ignore them, the framework will return STATUS_FLT_DO_NOT_ATTACH, and the filter won't receive any notifications.